### ISSUE TITLE
Error on Dashboard

### ISSUE THREAD

**Issue by tarantej** [hypecal/wordpress-events-manager-tickets-payments#9]
I am testing this plugin in my local wordpress installation. The plugin was throwing an error because it could not find the kint library (I resolved it but I request the developer to include it in the next version)

Now when the plugin is running I get this message on my dashboard

You've completed 2 of our list of 3 actions to activate the ROBBY plugin. Follow step by step each section and your tickets will be online in a minute!

![robbyticket](https://cloud.githubusercontent.com/assets/21302579/18041484/2ad1ee56-6dd7-11e6-8672-8725099d6af1.png)

What are the steps that are to be taken. Kindly include instructions in your next plugin update

I already have Events Manager and Events Manager ESS installed

**Comment by BricePissard** [comment]
Hi,

where did you get the Kint library error throwing, you shouldn't need it, it's a debugging tool.

The screepage you show me seems to have a javascript error, could you show me the content of the Chrome developper toolbar (View > Developper > Developper Tools : tab Console).

**Comment by tarantej** [comment]
This is what happens if i dont install kint library
![no kint](https://cloud.githubusercontent.com/assets/21302579/18299218/a7d5ec48-74dc-11e6-8852-26f56a55a12c.png)

**Comment by tarantej** [comment]
This is what happens if i dont install kint library
![no kint](https://cloud.githubusercontent.com/assets/21302579/18299218/a7d5ec48-74dc-11e6-8852-26f56a55a12c.png)


![no kint](https://cloud.githubusercontent.com/assets/21302579/18336460/ffafcd30-75a4-11e6-88bc-3ac9229596b2.png)

**Comment by tarantej** [comment]
Ok i got them
![robby missing js](https://cloud.githubusercontent.com/assets/21302579/18336794/f3d43322-75a7-11e6-9350-dd3509cbb0dd.png)

**Comment by BricePissard** [comment]
Ok, I've sent a new release to fix that issue. The problem was that under your localhost environment the script was acting as being under our developing environment, for security and code portability some elements are loaded through a WebService which in our dev is retrieve from http://localhost:8080

The version 1.4.2 fix that issue.

**Comment by tarantej** [comment]
These Errors are occurring even when the event has been published.  

![custom tickets not appearing](https://cloud.githubusercontent.com/assets/21302579/18658894/437b5826-7f24-11e6-8d2a-8020311ad9d0.png)
![error even when the event has been published](https://cloud.githubusercontent.com/assets/21302579/18658891/4028855e-7f24-11e6-813b-6f8a299bf736.png)
![event published](https://cloud.githubusercontent.com/assets/21302579/18658892/40431892-7f24-11e6-9ad7-4db5fd99db88.png)

**Comment by BricePissard** [comment]
I think I see where the problem comes from and it might be tricky to fix...

When a ticket is created, even in "Draft" mode, the data are duplicated on robby.ai database cloud servers to be able, once validated and online, to broadcast your tickets on other websites (to boost your sells, boost your visibility, SEO, etc..), but the problem here is that the script try to validate automatically the event sources (many parameters are toke into account from title, desc, img, geo loc...) and with localhost as domain name provider no binding IDs between your server and robby.ai server are made (a minima the source can't be trusted).

I'll dig into that issue, working under localhost is not something rare and making it working is a must.

The other problem you are facing with the empty "Custom Tickets" tab is more weird, could you post here the same Chrome JS Developer Tool screenshot.

**Comment by tarantej** [comment]
Yes absolutely. There must be many devs who believe in testing before deploying. They may be paranoid, but being paranoid and finding a problem is way better than finding the problem after the client website has crashed

![empty custom tickets](https://cloud.githubusercontent.com/assets/21302579/18663151/f520a598-7f3a-11e6-8278-054017a32ed4.jpg)

**Comment by BricePissard** [comment]
I made an update to work around your issue.

Could you update both the ESS Wordpress plugin and the ticketing plugin and tell me if it solves your localhost configuration issue?

**Comment by tarantej** [comment]
I dont think it worked. Are your plugins compatible with Wordpress version 4.6.1. I will try and reinstall all the plugins and see what happens

**Comment by tarantej** [comment]
The plugins wont even delete

**Comment by tarantej** [comment]
Same problem occurring in test subdomain

**Comment by tarantej** [comment]
Could this help you?
![error at subdomain](https://cloud.githubusercontent.com/assets/21302579/18698394/9b30784e-7fe7-11e6-8caf-0c304ffa2e1f.png)

**Comment by BricePissard** [comment]
The plugin is working in several environments and with the latest version of Wordpress 4.6.1 and Events Manager.

Not being able to delete a plugin can only have to see with your local configuration because it's the Wordpress software that handle it not the plugin.

I made some changes yesterday to be able to save events from a localhost environment and then create tickets on it, plus I had to reorder some cascading JS code that had to be loaded in a different order due to some Wordpress 4.6.1 changes.

But all that shouldn't impact your website as you've described upper.

I would really appreciate that all work for good for you, could you tell me what precisely is not working for you, where do you get stuck?

Debugging Wordpress plugin can be very tricky when you have to deal with so many environments from server to client browser...
